addons/jwt/src/JWTDecoder.gd

```diff
@@ -16,7 +16,15 @@ func _parse_json(field) -> Dictionary:
     var parse_result = JSON.new()
     if parse_result.parse(field) != OK:
         return {}
-    return parse_result.get_data()
+    var data = parse_result.get_data()
+    _maybe_pack_array(data, JWTClaims.Public.AUDIENCE)
+    return data
+ 
+ func _maybe_pack_array(dict, key):
```

Comment:
I believe that the space in front of this line is in error and is tripping up Godot when parsing this file.

Reply:
Thanks! Added #25